### ISSUE TITLE
6 of the 9 chatbots return errors

### ISSUE THREAD

**Issue by seanfitz64** [SingularityLabs-ai/MultiGPT-mini#34]
**Describe the bug**
When I do a multi-search 6 of the 9 chatbots return errors.

**To Reproduce**

Just input anything.

**Expected behavior**

Results.

**Screenshots**

![Image 052](https://github.com/SingularityLabs-ai/MultiGPT-mini/assets/169568966/5469cfa3-6645-4715-b254-02bf5ae340f8)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Chrome

**Additional context**
I'd really like to see this work.

**Comment by allentown521** [comment]
you can use aichatone.com for instead

**Comment by henmedia** [comment]
> you can use aichatone.com for instead

not really if limited to two bots in the free version :(

**Comment by allentown521** [comment]
> > you can use aichatone.com for instead
> 
> not really if limited to two bots in the free version :(

We have a three day free trial to see if it's worth it. not only the feature chat with multiple chatbots

**Comment by henmedia** [comment]
> We have a three day free trial to see if it's worth it. not only the feature chat with multiple chatbots

where can I find that?

**Comment by allentown521** [comment]
> > We have a three day free trial to see if it's worth it. not only the feature chat with multiple chatbots我们有三天的免费试用期，看看是否值得。不仅是与多个聊天机器人聊天的功能
> 
> where can I find that?我在哪里可以找到它？

upgrade to version 2.12.0 ,then you can find it here:
![QQ截图20240516090808](https://github.com/SingularityLabs-ai/MultiGPT-mini/assets/69662387/48e09496-68c2-4991-afc0-cb04cd312faf)

**Comment by henmedia** [comment]
> upgrade to version 2.12.0 ,then you can find it here:

Version 2.11.2 is the latest build in EDGE Store

**Comment by allentown521** [comment]
> > upgrade to version 2.12.0 ,then you can find it here:
> 
> Version 2.11.2 is the latest build in EDGE Store

yes, under reviewing on edge store,  you can install it via chrome web store:
https://chrome.google.com/webstore/detail/ejlkdnjeljojepglcdjifkedjbfdakbb

**Comment by seanfitz64** [comment]
> you can use aichatone.com for instead

So to be clear - you offer a free version that doesn't work, and when I point that out you send me to a paid version? I just want to be clear about the deceptive bait-and-switch that is going on here before I give your extension a 1-star bad review.

**Comment by henmedia** [comment]
> > you can use aichatone.com for instead
> 
> So to be clear - you offer a free version that doesn't work, and when I point that out you send me to a paid version? I just want to be clear about the deceptive bait-and-switch that is going on here before I give your extension a 1-star bad review.

It seems that it is not very trustworthy. No legal information on the website (who is offering this service, who wil get my money), possibly faked reviews, "5 stars from+2k happy customers" - no way

Anyway, the extension does it's job

**Comment by allentown521** [comment]
> > you can use aichatone.com for instead
> 
> So to be clear - you offer a free version that doesn't work, and when I point that out you send me to a paid version? I just want to be clear about the deceptive bait-and-switch that is going on here before I give your extension a 1-star bad review.

I noticed that this project does not seem to be maintained anymore, and we provide a similar experience. Chatting with multiple bots is just one of our features. It is indeed a paid value-added one. If the free version cannot meet all your needs, then I am Sorry, but does this have anything to do with deception? Free trial is a feature that many users have recently mentioned. We have implemented it. It does not require any credit card information from you, just an email address. It is impossible for me to explain these in detail under each of my replies, so I provide the official website address, where There are detailed instructions

**Comment by allentown521** [comment]
> > > you can use aichatone.com for instead
> > 
> > 
> > So to be clear - you offer a free version that doesn't work, and when I point that out you send me to a paid version? I just want to be clear about the deceptive bait-and-switch that is going on here before I give your extension a 1-star bad review.
> 
> It seems that it is not very trustworthy. No legal information on the website (who is offering this service, who wil get my money), possibly faked reviews, "5 stars from+2k happy customers" - no way
> 
> Anyway, the extension does it's job

It only represents the number of my users, not the number of 5-star users, or the icon is misleading

**Comment by seanfitz64** [comment]
> > > you can use aichatone.com for instead
> > 
> > 
> > So to be clear - you offer a free version that doesn't work, and when I point that out you send me to a paid version? I just want to be clear about the deceptive bait-and-switch that is going on here before I give your extension a 1-star bad review.
> 
> I noticed that this project does not seem to be maintained anymore, and we provide a similar experience. Chatting with multiple bots is just one of our features. It is indeed a paid value-added one. If the free version cannot meet all your needs, then I am Sorry, but does this have anything to do with deception? Free trial is a feature that many users have recently mentioned. We have implemented it. It does not require any credit card information from you, just an email address. It is impossible for me to explain these in detail under each of my replies, so I provide the official website address, where There are detailed instructions

Okay, so MultiGPT is defunk and you are from another service. I thought you were from MultiGPT.

**Comment by ishandutta2007** [comment]
@seanfitz64  I am from team MultiGPT , please don't reply to other advertisements on this thread. 

> I noticed that this project does not seem to be maintained anymore,

Our production version is not open-source, so you will not see changes on github. The open-source version is older and smaller subset. The version you are using (v0.0.32) is latest and have been updated last week only, so it should work as long as you have followed the istructions.

As far as your issue we don't see any issues from our end with OS: Windows 11 or Browser: Chrome. 
1) Can you verify that each bot is working directly? Hope you have read the instruction that you need to be logged into each of those websites. You can close the tab but Do not logout from any of them.
2) In case the issue still exists we would like to know your Country of residence as well.

**Comment by seanfitz64** [comment]
> @seanfitz64 I am from team MultiGPT , please don't reply to other advertisements on this thread.
> 
> > I noticed that this project does not seem to be maintained anymore,
> 
> Our production version is not open-source, so you will not see changes on github. The open-source version is older and smaller subset.
> 
> As far as your issue we don't see any issues from our end with OS: Windows 11 or Browser: Chrome. So we would like to know your Country, and verification that it is working directly.

I am in Australia and I don't know what you mean by "verification that it is working directly".

**Comment by ishandutta2007** [comment]
@seanfitz64  Hey we found some of the issues(except POE ones), will update you soon. Thanks for creating the issue.  We just took our eye off for a week and so many stuff broke(you can check out last release was only few days back).

**Comment by henmedia** [comment]
Is there any development going on? This is falling behind other solutions, still no GPT4o for example.

**Comment by ishandutta2007** [comment]
@henmedia can you tell us a list , will prioritise accordingly.

**Comment by henmedia** [comment]
for me it's GPT 4o and Claude 3.5

**Comment by henmedia** [comment]
2 weeks later, still nothing happened - is this project dead?

**Comment by MauryaAnurag** [comment]
deaddddddddd
